stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -54,17 +54,11 @@ set(SOURCES
 set(GYB_SOURCES
   ExecuteOp.swift.gyb)
 
-# Copy TensorFlow bindings file, if it exists.
-if (TENSORFLOW_SWIFT_BINDINGS)
-  file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES
-    "${TENSORFLOW_SWIFT_BINDINGS}/*.swift")
-  list(APPEND SOURCES "${TENSORFLOW_SWIFT_BINDINGS_SOURCES}")
-endif()
-
 # Copy TensorFlow high-level API sources, if they exist.
 if (TENSORFLOW_SWIFT_APIS)
   file(GLOB_RECURSE TENSORFLOW_SWIFT_API_SOURCES
     "${TENSORFLOW_SWIFT_APIS}/Sources/*.swift"
+    "${TENSORFLOW_SWIFT_APIS}/Sources/Bindings/*.swift"
     "${TENSORFLOW_SWIFT_APIS}/Sources/Core/*.swift")
```

Comment:
A [`Layers`](https://github.com/tensorflow/swift-apis/tree/master/Sources/DeepLearning/Layers) directory got added to swift-apis recently. Could you help to add that here too?

Reply:
I actually didn't understand why we added the `Core` directory in the first place. I think we only need `"${TENSORFLOW_SWIFT_APIS}/Sources/*.swift"`. That's actually why I was using `GLOB_RECURSE` instead of `GLOB` but `"${TENSORFLOW_SWIFT_APIS}/Sources/Core/*.swift"` was added at some point.

Reply:
I believe it was added by https://github.com/tensorflow/swift-apis/pull/133. I don't think `Dataset.swift` should be in a `Core/` directory.

Reply:
In tensorflow/swift-apis#109 I have put `Dataset.swift` in the `Operations` directory because it mainly contains TF op wrappers. In either case though, since we're using `GLOB_RECURSE` we should only need `"${TENSORFLOW_SWIFT_APIS}/Sources/*.swift"` no matter what subdirectories exist in `Sources`. I can remove the `Bindings` and the `Core` lines if you want. Tests seem to work fine on my machine without them.

Reply:
Removing those lines sounds good.

Reply:
Done.